pkg/cmd/cli/client/config/get.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 the Heptio Ark contributors.
```

Comment:
2018

Reply:
This was actually a move from a previous file, but I can adjust it.